client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -238,6 +224,8 @@ class AdsWrapper extends Component {
 			);
 		} else if ( ! site.options.wordads && isWordadsInstantActivationEligible( site ) ) {
 			component = this.renderInstantActivationToggle( component );
+		} else if ( ! canAccessAds( site ) ) {
+			component = this.renderEmptyContent();
```

Comment:
@Aurorum shouldn't this be the first condition to check in the `if` chain? Even before potentially displaying the instant activation toggle?

Reply:
Switched the logic around so that it's the first condition to check. 👍